### PR TITLE
Remove utils package

### PR DESCRIPTION
**Pull request by peterszerzo**
This PR removes the utils package from the project, in the following order of commits:
* [x] remove from the components package
* [x] remove from the visualizations package
* [x] remove from visual tests
* [x] remove utils package itself

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,15 +39,8 @@
...
-    "*.(js|jsx|json|html|md)": [
-      "prettier --write",
-      "git add"
-    ],
-    "*.(ts|tsx)": [
-      "tslint --fix",
-      "prettier --write",
-      "git add"
-    ]
+    "*.(js|jsx|json|html|md)": ["prettier --write", "git add"],
```
Comment (fabien0102) [inline package.json]: Prettier doesn't fix html ;)

File: packages/utils/src/__tests__/text.test.ts [hunk 2]
```diff
@@ -1,16 +0,0 @@
-import { getInitials } from "../text"
```
Comment (fabien0102) [inline packages/utils/src/__tests__/text.test.ts]: If you remove my unit tests, I will not be happy ^^

**Comment by fabien0102** [comment]
Travis got the truth, no `prettier` inside our `devDependencies` 😉

**Comment by peterszerzo** [comment]
@contiamo/frontend tests are passing again :)

**Pull request merged**